### PR TITLE
Reader Refresh: disambiguate rec clicks for recording tracks

### PR DESCRIPTION
**Pull request by samouri**
Add extra props to recommendation_clicked tracks event #9762

#### To test:
- [x] ensure that the events **calypso_reader_recommended_site_clicked** is activated from both search page and in-stream recs
- [x] ensure that the events **calypso_reader_recommended_post_clicked** is activated from both search page and in-stream recs are clicked

### CODE REVIEW

File: client/reader/stats.js [hunk 1]
```diff
@@ -116,6 +116,7 @@ tracksRailcarEventWhitelist
 	.add( 'calypso_reader_searchcard_clicked' )
 	.add( 'calypso_reader_author_link_clicked' )
 	.add( 'calypso_reader_permalink_click' )
+	.add( 'calypso_reader_recommended_post_clicked' )
```
Comment (bluefuton) [inline client/reader/stats.js]: Need to add `calypso_reader_recommended_site_clicked` here too.

Comment (samouri) [inline client/reader/stats.js]: oh right! thanks

Comment (bluefuton) [inline client/reader/stats.js]: Added in d6788db 👍

**Comment by samouri** [comment]
awesome!  Thanks for the assist @bluefuton

**Pull request merged**